lntest/itest/assertions.go

```diff
@@ -615,9 +604,7 @@ func shutdownAndAssert(net *lntest.NetworkHarness, t *harnessTest,
 	err := wait.NoError(func() error {
 		return net.ShutdownNode(node)
 	}, defaultTimeout)
-	if err != nil {
-		t.Fatalf("unable to shutdown %v: %v", node.Name(), err)
-	}
+	require.NoError(t.t, err, "unable to shutdown %v", node.Name())
```

Comment:
Needs to be `NoErrorf` for the `%v` to be interpreted.

Reply:
Fixed!